src/backend/src/utils/slack.utils.ts

```diff
@@ -67,3 +67,18 @@ export const sendSlackTaskAssignedNotification = async (slackId: string, task: T
   const linkButtonText = 'View Task';
   await sendMessage(slackId, msg, link, linkButtonText);
 };
+
+/**
+ * Send a notification to users that reimbursement request is denied on Slack
+ * @param slackId the slack id of the assignee
+ * @param denial the denial if the reimbursement request
+ */
+export const sendReimbursementRequestDeniedNotification = async (slackId: string, requestId: string): Promise<void> => {
+  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
+
+  const msg = `Your reimbursement request has been denied.`;
+  const link = `https://finishlinebyner.com/finance/reimbursement-requests/${requestId}`;
+  const linkButtonText = 'View Reimbursement Request';
+
+  await sendMessage(slackId, msg, link, linkButtonText);
```

Comment:
add a try catch around sendMessage, otherwise if it fails it'll crash the backend

Reply:
are you looking for something like 
try {
    await sendMessage(slackId, msg, link, linkButtonText);
  } catch (error: unknown) {
    if (error instanceof Error) {
      console.error(error);
    }
  }

Reply:
yeah but the catch should throw an HTTPException which is an error our backend can parse (we do it in a couple other places)

Reply:
so it should be this "throw new HttpException(500, `Failed to send slack notification: ${error.message}`);" instead of this "console.error(error);" Also what should the else be since it is an if statement? It's like if blah else blah blah right so what is an acceptable else statement?

Reply:
yeah, you don't need the else for this one, the if is just to make sure the error data is in a format we can use